addons/event/tests/test_event_mail_schedule.py

```diff
@@ -341,3 +341,53 @@ def test_archived_event_mail_schedule(self):
 
         # check that scheduler is not executed
         self.assertFalse(event_prev_scheduler.mail_done, 'event: reminder scheduler should should have run')
+
+    @mute_logger('odoo.addons.base.models.ir_model', 'odoo.models')
+    def test_include_mails_to_unconfirmed_attendees_in_the_sent_count(self):
```

Comment:
maybe "test_event_mail_before_trigger_sent_count" or something like that and test more than just "unconfirmed"